ci/run_easymesh_cert.sh

```diff
@@ -0,0 +1,115 @@
+# See LICENSE file for more details.
+###############################################################
+
+#!/bin/bash
+# Simple wrapper script to easymesh_cert/run_test_file.sh
+# which also updates to the latest prplmesh IPK, run the tests and
+# uploads the results
+
+scriptdir="$(cd "${0%/*}"; pwd)"
+
+dbg() {
+    if [ "$VERBOSE" = "true" ]; then echo "$(basename $0): $*"; fi
+}
+
+run() {
+    dbg "$*"
+    "$@"
+}
+
+err() {
+    printf '\033[1;31m%s\033[0m\n' "$*"
+}
+
+info() {
+    printf '\033[1;35m%s: %s\033[0m\n' "$(basename "$0")" "$*"
+}
+
+success() {
+    printf '\033[1;32m%s\033[0m\n' "$*"
+}
+
+usage() {
+    echo "usage: $(basename $0) [-hboev] <test> [test]"
+    echo "  options:"
+    echo "      -h|--help - display this help."
+    echo "      -b|--branch - prplmesh branch to use (default master latest)"
+    echo "      -o|--log-folder - path to put the logs to (make sure it does not contain previous logs)."
+    echo "      -e|--easymesh-cert - path to easymesh_cert repository (default ../easymesh_cert)"
+    echo "      -u|--upload - upload results"
+    echo "      -v|--verbose - set verbosity (ucc logs also redirected to stdout)"
+    echo "'test' can either be a test name, or the path to a file containing"
+    echo " test names (newline-separated)."
+    echo
+    echo "Multiple test files/names are allowed, and they can also be mixed."
+    echo "For example, the following call would run the test 'MAP-5.3.1', and all the tests"
+    echo "contained in the 'tests/all_controller_test.txt' file:"
+    echo "$(basename "$0") MAP-5.3.1 tests/all_controller_test.txt"
+    echo ""
+    echo "The default is to run all agent certification tests."
+}
+
+main() {
+     OPTS=$(getopt -o 'hb:o:e:u:v' --long help,branch:,log-folder:,easymesh-cert:,upload:,verbose -n 'parse-options' -- "$@")
+
+    if [ $? != 0 ] ; then echo "Failed parsing options." >&2 ; usage; exit 1 ; fi
+
+    eval set -- "$OPTS"
+
+    while true; do
+        case "$1" in
+            -h | --help)            usage; exit 0;;
+            -b | --branch)          BRANCH="$2"; shift 2;;
+            -o | --log-folder)      LOG_FOLDER="$2"; shift 2;;
+            -v | --verbose)         VERBOSE=true; VERBOSE_OPT="-v"; shift;;
+            -e | --easymesh-cert)   EASYMESH_CERT_PATH="$2"; shift 2;;
+            -u | --upload)          UPLOAD_PATH="$2"; shift 2;;
+            -- ) shift; break ;;
+            * ) err "unsupported argument $1"; usage; exit 1;;
+        esac
+    done
+
+    TESTS="$@"
+
+    [ -n "$TESTS" ] || TESTS="$EASYMESH_CERT_PATH/tests/all_agent_tests.txt"
+    [ -n "$LOG_FOLDER" ] || LOG_FOLDER="$EASYMESH_CERT_PATH/logs/$(date +%F_%H-%M-%S)"
+    info "Logs location: $LOG_FOLDER"
+    info "Tests to run: $TESTS"
+
+    info "download latest ipk"
+    run "$PRPLMESH_PATH"/tools/download_ipk.sh --branch "$BRANCH"
+
+    info "deploy latest ipk"
+    run "$PRPLMESH_PATH"/tools/deploy_ipk.sh "$USER"@"$IPADDR" "$IPK"
```

Comment:
Given that those two scripts are there, I think it's more appropriate to put this run_easymesh_cert script in the same directory. (nitpick)

Also I think it's worth to make a variable for the tools directory.

Reply:
@rmelotte preferred to have a new folder `ci` since we will probably have more scripts related to CI in the future, so I left it as is. I did use a variable for the tools directory, however.

Reply:
We could arguably move `ci` to `tools/ci`, but yes I think it makes sense to group the scripts related to CI in the same folder (we definitely have more to come).